### PR TITLE
Added doxygen script for wifi manager

### PR DESCRIPTION
**Pull request by JeonginKim**

### CODE REVIEW

File: framework/include/wifi_manager/wifi_manager.h [hunk 1]
```diff
@@ -23,6 +23,11 @@
  * @{
  */
 
+/**
+ * @file wifi_manager.h
+ * @brief Provides APIs for Wi-Fi Manager
+ */
```
Comment (sunghan-chang) [inline framework/include/wifi_manager/wifi_manager.h]: Don't you need to set group?

Comment (jeongchanKim) [reply framework/include/wifi_manager/wifi_manager.h]: I think, in the above, there is a { for group, so ingroup is not need.

Comment (sunghan-chang) [reply framework/include/wifi_manager/wifi_manager.h]: Oh ok. Then, why do you repeat brief?

Comment (sunghan-chang) [reply framework/include/wifi_manager/wifi_manager.h]: Let's move file option to above comments and remove duplicated brief.

**Pull request merged**